src/writesbml.jl

```diff
@@ -777,6 +508,13 @@ function _create_doc(mdl::Model)::VPtr
     return doc
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+Write the SBML structure in `mdl` to a file `fn`.
+
+To write the XML to a string, use the other overload.
```

Comment:
Maybe make it more explicit like

> To write the XML to a string, use `writeSBML(mdl::Model)`

?  Likewise for the other docstring.

Reply:
true, thanks